webpack.config.js

```diff
@@ -487,13 +487,6 @@ if (isDev || isTest) {
             'process.env': {
                 'NODE_ENV': `"${process.env.NODE_ENV || 'production'}"`
             }
-        }),
-        new webpack.optimize.UglifyJsPlugin({
```

Comment:
this is removing uglification in production build, is it not?  it's possible btw that e2e doesn't build in "test" mode.  but in that case we need to fix that.

Reply:
Ah, I accidentally committed this change to the branch. It solves the 1hr build time for the frontend on my machine and jitpack. I have reverted these lines back to rc state.  This problems still needs to be solved though...